performance-es.md

```diff
@@ -57,3 +57,117 @@ La optimización prematura también puede afectarte al atarte a ciertas decision
 En la gran mayoría de casos, el tamaño y velocidad del programa no es el problema. La optimización más fácil es no hacerla. La segunda alternativa más fácil es simplemente comprar mejor hardware.
 
 Cuando hayas decidido cambiar tu programa, sigue leyendo.
+
+### Modificar los datos
+
+Modificar los datos significa agregar o alterar la representación de los datos
+que estas procesando. Desde el punto de vista del rendimiento, algunos de
+estos acabaran cambiando la complejidad O() asociada a diferentes aspectos de
+la estructura de datos.
+
+Ideas para mejorar tu estructura de datos:
+
+* Campos adicionales
+
+  El clásico ejemplo de esto es almacenar la longitud de una lista enlazada en un
+  campo en el nodo raíz. Mantenerla actualizada lleva un poco más de trabajo,
+  pero consultar la longitud se convierte en un simple acceso a un campo en vez de una
+  búsqueda transversal con complejidad O(n). Tu estructura de datos puede presentar una
+  mejora similar: un poco de mantenimiento en algunas operaciones a cambio de
+  mejorar el rendimiento en un caso de uso común.
```

Comment:
Since we are talking about a win regarding an operation to improve performance of a `common` or `frequent` case.
`s/en un caso de uso común/en casos frecuentes/'

Reply:
I feel like both say pretty much the same, but `caso de uso comun` is a more literal translation of the original sentence. Maybe we should have a third opinion on this? @moncho

Reply:
Between the two options, I would go for the literal translation here.